capa/features/extractors/ida/helpers.py

```diff
@@ -90,8 +90,10 @@ def get_file_imports() -> Dict[int, Tuple[str, str, int]]:
         if not library:
             continue
 
-        # IDA uses section names for the library of ELF imports, like ".dynsym"
-        library = library.lstrip(".")
+        # IDA uses section names for the library of ELF imports, like ".dynsym".
+        # These are not useful to us, we may need to expand this list over time (TODO: exhaust this list)
+        if library == ".dynsym":
+            library = ""
```

Comment:
agree that this is a good start. lets open a new issue to track the "section name as library name" in IDA issue.

Reply:
opened issue #1419